crates/proof-of-sql/src/base/database/owned_table_utility.rs

```diff
@@ -55,7 +55,7 @@ pub fn owned_table<S: Scalar>(
 /// ]);
 ///```
 /// # Panics
-/// - Panics if `name.parse()` fails to convert the name into an `Identifier`.
+/// - Panics if `name.parse()` fails to convert the name into an `Ident`.
```

Comment:
This shouldn't panic any more. Also we can't run `name.parse()` any more either. Same for other cases.